docker/tox-Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM ubuntu:14.04.5
+WORKDIR /app
+
+RUN apt-get update && apt-get install -y \
+    curl \
+    make \
+    ghostscript \
```

Comment:
Ditto, do we need all these libraries for this repository?
I see dependencies that are used for fonts and images, this library is only used to handle database stuff, please review and remove unnecessary libraries.